components/CourseContentVideoNavSide.vue

```diff
@@ -8,6 +8,7 @@
       <v-row class="pa-0 ma-0">
         <v-col cols="12" md="8" class="py-0">
           <div class="subheader video-title">{{video.name}}</div>
+          <div v-if="video.pro && proVideoInFreePeriod(video) && userNotPro" class="pro-time-left green--text text--accent-3">{{ timeLeft(video, currentTime) | formatTimeLeft }}</div>
```

Comment:
This is something that's long enough that it would benefit from multiple lines.

```html
<div v-if="video.pro && proVideoInFreePeriod(video) && userNotPro"
     class="pro-time-left green--text text--accent-3">
  {{ timeLeft(video, currentTime) | formatTimeLeft }}
</div>
```